### PR TITLE
First pr

### PR DESCRIPTION
**Pull request by d3fkn1ght**
﻿## Description of Change

Updating comments to fix CS 

### Bugs Fixed

None, just updated comments to fix warnings

### CODE REVIEW

File: Source/Xamarin/Prism.Forms/Navigation/INavigationService.cs [hunk 1]
```diff
@@ -39,7 +38,7 @@ public interface INavigationService
         /// <param name="parameters">The navigation parameters</param>
         /// <remarks>Navigation parameters can be provided in the Uri and by using the <paramref name="parameters"/>.</remarks>
         /// <example>
-        /// Navigate(new Uri("MainPage?id=3&name=brian", UriKind.RelativeSource), parameters);
+        /// Navigate(new Uri("MainPage?id=3&amp;name=brian", UriKind.RelativeSource), parameters);
```
Comment (brianlagunas) [inline Source/Xamarin/Prism.Forms/Navigation/INavigationService.cs]: Should be "NavigateAsync"

File: Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs [hunk 2]
```diff
@@ -279,7 +278,7 @@ public virtual Task<INavigationResult> NavigateAsync(Uri uri)
         /// <param name="parameters">The navigation parameters</param>
         /// <remarks>Navigation parameters can be provided in the Uri and by using the <paramref name="parameters"/>.</remarks>
         /// <example>
-        /// Navigate(new Uri("MainPage?id=3&name=brian", UriKind.RelativeSource), parameters);
+        /// Navigate(new Uri("MainPage?id=3&amp;name=brian", UriKind.RelativeSource), parameters);
```
Comment (brianlagunas) [inline Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs]: Example should be NavigateAsync

File: Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs [hunk 3]
```diff
@@ -300,7 +299,7 @@ Task<INavigationResult> IPlatformNavigationService.NavigateAsync(Uri uri, INavig
         /// <param name="animated">If <c>true</c> the transition is animated, if <c>false</c> there is no animation on transition.</param>
         /// <remarks>Navigation parameters can be provided in the Uri and by using the <paramref name="parameters"/>.</remarks>
         /// <example>
```
Comment (brianlagunas) [inline Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs]: No need to provide and example on this method as this is a protected method only meant for implementation by derived classes.

File: Source/Xamarin/Prism.Forms/Navigation/INavigationService.cs (hunk 1 above)
Comment (d3fkn1ght) [reply Source/Xamarin/Prism.Forms/Navigation/INavigationService.cs]: Interesting.  I'll modify.  That is the code that came from the Prism repo.  I only modifed the &amp;

I'll modify it

**Pull request merged**